<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-UQLJLBQ3" failures="1" tests="8" name="Tests.TestCases_UI_3" time="31.380" errors="0" timestamp="2025-09-17T14:09:13 IST" skipped="0">
  <testcase classname="Tests.TestCases_UI_3" name="verifySubscriptionFieldIsAcceptingInput" time="3.864"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifyFeedbackForUsFontSizeIsCorrect" time="3.148"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifySubscriptionButtonIsWorkingGood" time="3.758"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifyFeedbackForUsFontStyleIsCorrect" time="3.359"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifyEmailIdURLForFeedback" time="3.621">
    <failure message="The Email ID URL for feedback was found on the Test Cases page when it should not be present." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The Email ID URL for feedback was found on the Test Cases page when it should not be present.
at org.testng.Assert.fail(Assert.java:111)
at Tests.TestCases_UI_3.verifyEmailIdURLForFeedback(TestCases_UI_3.java:161)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:569)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- verifyEmailIdURLForFeedback -->
  <testcase classname="Tests.TestCases_UI_3" name="verifyContentsUnderFeedbackForUs" time="4.012"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifyFeedbackForUsTitleIsPresentInCorrectPosition" time="3.859"/>
  <testcase classname="Tests.TestCases_UI_3" name="verifyFunctionalityOfScrollToTopButton" time="5.759"/>
</testsuite> <!-- Tests.TestCases_UI_3 -->
